libbeat/template/load.go

```diff
@@ -247,7 +205,8 @@ func (b *templateBuilder) buildBody(tmpl *Template, config TemplateConfig, field
 		return b.buildBodyFromFile(tmpl, config)
 	}
 	if fields == nil {
-		return b.buildMinimalTemplate(tmpl)
+		b.log.Debug("Load minimal template")
```

Comment:
minimal template = standard template without any modifications?

Reply:
Standard template without fields. This can be probably removed as all Beats have fields and we no longer support building community Beats.